packages/app-store/umami/config.json

```diff
@@ -14,7 +14,7 @@
     "tag": {
       "scripts": [
         {
-          "src": "https://us.umami.is/script.js",
+          "src": "{SCRIPT_URL}",
```

Comment:
I think it's better to let the user choose whether they want to add SCRIPT_URL or not

Reply:
Looks good